vendor/engines/bulk_email/app/models/bulk_email/job.rb

```diff
@@ -0,0 +1,21 @@
+module BulkEmail
+
+  class Job < ActiveRecord::Base
+
+    self.table_name = "bulk_email_jobs"
+
+    belongs_to :facility, foreign_key: :facility_id
+
+    serialize :recipients, Array
+    serialize :search_criteria, Hash
```

Comment:
This serializes to YAML, correct? I don't have a problem with that; just want to confirm.